### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by hamaadtahiir** [jokerlc/RMGN-VITON#4]
Kindly provide the training code

**Comment by ansarker** [comment]
@jokerlc planning to provide training code?

**Comment by MaksMaib** [comment]
Hi, there. Does i understand correct, that you have the same train loop like in PF-AFN, the only difference in last step, where you use your mask guided module?
Thanks for you work and answer.

**Comment by jokerlc** [comment]
yes, the biggest difference is the PF-Generator with our mask guided module; another difference is the postrue awareness loss

**Comment by MaksMaib** [comment]
> 
Thank you, for rapid and detailed answer. You rock!

**Comment by ansarker** [comment]
@jokerlc for posture awareness loss, what data you used, I mean densepose or openpose?
